pkg/derivation/encode.go

```diff
@@ -214,7 +212,7 @@ func (d *Derivation) writeDerivation(
 		}
 	}
 
-	_, err := writer.Write([]byte("Derive"))
+	_, err := writer.Write(derivationPrefix)
```

Comment:
That's unrelated. Separate commit?

Reply:
The motivation was to remove an allocation, it's related.